Apps/Sandcastle/gallery/Time Dynamic Point Cloud.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Time Dynamic Point Cloud">
+    <meta name="cesium-sandcastle-labels" content="Showcases, 3D Tiles">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    shouldAnimate : true
+});
+
+var basePath = '../../SampleData/test/PointCloudTimeDynamic/';
+var czmlPath = basePath + 'frames.czml';
+var dataSource = viewer.dataSources.add(Cesium.CzmlDataSource.load(czmlPath));
+dataSource.then(function(dataSource) {
+    var entities = dataSource.entities;
+    var entity = entities.getById('Time Dynamic Point Cloud');
+    var properties = entity.properties;
+    var frames = properties.frames;
```

Comment:
Should the CZML spec be updated?

Reply:
Hardcoding sounds OK; if CZML can now point to a .pnts file, we should update the spec for that.

Reply:
No CZML spec updates required.

The demo is just using CZML as a storage mechanism and helper.

There was some offline discussion that this use case is too niche to be added to core CZML, so I never built that in. But the demo shows that if you had a czml with a custom property you could get a `TimeIntervalCollection` and initialize a `TimeDynamicPointCloud`. You can do this without CZML too, which is what I plan on doing in this demo instead.

Reply:
OK, I suppose.  We have someone relying on this and we need to provide them guidance on the best practice whether that be a custom CZML property or a completely separate container.  They are already a CZML user as you know but that does not imply that the point clouds needs to be part of it.